resource/resource.go

```diff
@@ -22,6 +22,6 @@ package resource
 // When changing files included here, you may add the `-debug` flag, which will
 // avoid embedding any files (but instead passes them through from disk). This
 // avoids having to recompile before testing changes.
-//go:generate go-bindata -pkg resource -mode 0644 -modtime 1400000000 -o ./embedded.go ./ui/...
+//go:generate go-bindata -pkg resource -mode 0644 -modtime 1400000000 -o ./embedded.go ./ui/... ./test_certs/...
```

Comment:
Aren't they optimized away in the main build? But of course I can just move them over.